extensions/sql-bindings/package.json

```diff
@@ -59,6 +59,10 @@
     "tslint": "^5.8.0",
```

Comment:
build 1960252276 needs to b undonw (ave pull test psv. #19306)

#19396 "tslint" psv. for then while
{fast-glob} {5.8.0.} #19396